sched/misc/assert.c

```diff
@@ -174,7 +473,20 @@ void _assert(FAR const char *filename, int linenum)
 
   syslog_flush();
 
-  up_assert(filename, linenum);
+  /* Show back trace */
+
+#ifdef CONFIG_SCHED_BACKTRACE
+  sched_dumpstack(running_task()->pid);
+#endif
+
+  up_assert();
```

Comment:
Seems wrong here, the dumped stack in showstacks() is different to the stack when register dumped in up_assert() due the function call and exit, which causes the stack changed. @xiaoxiang781216 @zyfeier

Reply:
What are the further actions here?

Reply:
@zyfeier is looking at this problem.

Reply:
> What are the further actions here?

@pkarashchenko here is the fix: https://github.com/apache/nuttx/pull/7955